mysite/app/templates/app/tilbud_liste_bookingansvarlig.html

```diff
@@ -0,0 +1,34 @@
+{% extends '../base.html' %}
+
+{% load static %}
+{% block styles %}
+    <link rel="stylesheet" href="{% static 'css/base.css' %}" type="text/css">
+{% endblock %}
+
+{% block content %}
+<div class="page-container">
+    <h1>Oversikt over godkjente tilbud</h1>
+    <h3>For å få mer informasjon om et tilbud og eventuelt sende, klikk på se mer</h3>
```

Comment:
Denne burde vekk da det er innlysende hva knappen vil gjøre hvis du kaller den for se detaljer.